run/test/specs/utils/capabilities_android.ts

```diff
@@ -4,7 +4,7 @@ import { W3CCapabilities } from "@wdio/types/build/Capabilities";
 import { isNil, isString } from "lodash";
 import { CapabilitiesIndexType } from "./capabilities_ios";
 
-const androidAppFullPath = `/Users/emilyburton/Documents/session-android/app/build/outputs/apk/play/debug/session-1.18.2-universal.apk`;
+const androidAppFullPath = `/Users/emilyburton/Downloads/session-android-20240531T000357Z-b544961d2-universal/session-1.18.4-universal.apk`;
```

Comment:
this shouldn't be commited but using a process.env variable

Reply:
Added an env file to moved both ios and android paths to that